django_netjsonconfig/base/admin.py

```diff
@@ -42,6 +42,7 @@ class Media:
               for f in ('preview.js',
                         'unsaved_changes.js',
                         'uuid.js',
+                        'template_admin.js',
```

Comment:
you should load this JS only in the template admin class, otherwise it will be loaded also where is not relevant